### ISSUE TITLE
WebGL errors when changing text length

### ISSUE THREAD

**Issue by arjanfrans** [Jam3/three-bmfont-text#3]
I created a text geometry and added it to a mesh with text value `1`. When I update the geometry and the text increases in length (for example `10`), then I get the following warning (using Google Chrome):

```
[GroupMarkerNotSet(crbug.com/242999)!:D036DE9CC4000000]GL ERROR :GL_INVALID_VALUE : glBufferSubData: out of range
[GroupMarkerNotSet(crbug.com/242999)!:D036DE9CC4000000]GL ERROR :GL_INVALID_OPERATION : glDrawElements: range out of bounds for buffer
```

The text then diappears.

If I understand correctly, three.js can not resize the buffer of a geometry. I am able to work around this by setting the initial text with a large value (for example `111111111`).

Is this a limitation of WebGL / three.js, or can this be fixed within three-bmfont-text?

**Comment by mattdesl** [comment]
This is definitely not a limitation of WebGL, but it might be a problem with ThreeJS.

I believe (though haven't yet confirmed) that the latest version of ThreeJS has some improvements to BufferGeometry which *should* allow us to grow buffers.

However, this module is still using an older version of ThreeJS - see #2.

In the next little while I'll try to see about upgrading this module.

**Comment by mattdesl** [comment]
This should be fixed in `2.0.0` but be sure to check the [change log](https://github.com/Jam3/three-bmfont-text#change-log) as there are some breaking changes.
